app/views/shared/_service_access.slim

```diff
@@ -1,7 +1,8 @@
 p
   ' You don't have access to any API on the
   => current_account.org_name
-  ' account. Please
-  - admin = current_account.admin_user
-  => mail_to admin.email, "contact #{current_account.admin_user_display_name}"
-  | to request access.
+  ' account.
+  - if admin = current_account.admin_user
+    ' Please
+    => mail_to admin.email, "contact #{current_account.admin_user_display_name}"
+    ' to request access.
```

Comment:
IMO you should change this line:
https://github.com/3scale/porta/blob/9751de00cd7e41820833ea2906b44d376227fb23/app/decorators/account_decorator.rb#L4

For:
```ruby
delegate :display_name, :email, to: :admin_user, prefix: true
```

And then this should become:
```suggestion
  ' account. Please
    => mail_to current_account.admin_user_email, "contact #{current_account.admin_user_display_name}"
    ' to request access.
```


For 2 reasons:

1. That `if` here is a problem. The account **must** have and admin_user at this point 😕 
2. Law of Demeter

Reply:
Will this fix https://app.circleci.com/pipelines/github/3scale/porta/14574/workflows/c20b5833-d424-44c5-9541-61f4064adab4/jobs/191319?

Reply:
oki i will check 👍🏼

Reply:
https://github.com/3scale/porta/pull/2246 @josemigallas